modules/compliance-profiles.adoc

```diff
@@ -214,18 +214,4 @@ title: Record Attempts to Alter Logon and Logout Events
 warning: Manual editing of these files may indicate nefarious activity, such as an
   attacker attempting to remove evidence of an intrusion.
 ----
-====
```

Comment:
This content was moved to it's own module so that it can be located two places within the docs for easy reference.